spec/takeaway_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'takeaway'
+
+describe TakeAway do
+
+  subject(:takeaway) {described_class.new()}
+  # let(:menu) { double(menu:, :pizza => 10 }
+
+  context 'Customer' do
+    it 'can see the menu' do
+      expect(takeaway).to respond_to :order
+        end
+
+    it 'can\'t order a dish not in the menu' do
+      # allow(menu).to receive(:dishes){ pasta, 10 }
+      takeaway()
+      expect{takeaway.add_meal(:pomodoro, 2)}.to raise_error 'dish not in the menu'
+    end
+    it 'can order a dish in the menu' do
+
+      expect{takeaway.add_meal(:pizza, 2)}.to change {takeaway.order.length}.by(1)
+    end
+    it 'can\'t delete a dish not in the ordered dishes' do
+      # allow(menu).to receive(:dishes){ pasta, 10 }
+      takeaway()
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments. (https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)